crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -0,0 +1,103 @@
+use clap::Parser;
+use notify::{RecursiveMode, Watcher};
+use std::path::{Path, PathBuf};
+use tokio::time;
+
+use crate::commands::build;
+
+use super::build::build_clients::LoamEnv;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub build_cmd: build::Cmd,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Watcher(#[from] notify::Error),
+    #[error(transparent)]
+    Build(#[from] build::Error),
+    #[error("IO error: {0}")]
+    Io(#[from] std::io::Error),
+}
+
+impl Cmd {
+    pub async fn run(&self) -> Result<(), Error> {
+        // Set LOAM_ENV to development
+        std::env::set_var("LOAM_ENV", "development");
+        let (tx, mut rx) = tokio::sync::mpsc::channel(100);
+        let tx_clone = tx.clone();
```

Comment:
What errors were you getting if this wasn't cloned?

Reply:
It works if it isn't cloned. There would be  a problem if tx was used later in the code but since it isn't I don't think we need to clone it.